### DISCUSSION TITLE
Coachmark - innsikt og tilbakemeldinger

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#2215] [Komponenter]
### Bli med å teste Coachmarks 🤩

Adobe beskriver Coachmarks slik:

> Coach marks are temporary messages that educate users through new or unfamiliar product experiences. They can be chained into a sequence to form a tour.

Vi har lagd et utgangspunkt til Coachmark og publisert komponentene i [Alpha UI kit](https://www.figma.com/file/Jw3DCrpJgK0uiG7v92Wdnk/Alpha-UI-kit?type=design&node-id=1891-453&mode=design). Vi ønsker at dere som har behov for denne komponenten tester den og melder tilbake innsikt.

**Comment by KenAJoh** [comment]
[Atlassian](https://atlassian.design/components/onboarding/examples)
